src/graphql/admin/schema.graphql

```diff
@@ -84,6 +85,29 @@ Date field, serialized as the number of milliseconds since the Unix Epoch
 """
 scalar Timestamp
 
+type LightningPayment {
+  status: String
```

Comment:
Potentially could use a type here similar to how we have `PaymentSendResult` and `InvoicePaymentStatus`

Reply:
added https://github.com/GaloyMoney/galoy/pull/613/commits/d8a70c3575b00c9fce0479cd981e01e52c3d4b0b